### PR TITLE
Updated the polyfit function to np.polynomial.Polynomial.fit()

### PR DESCRIPTION
**Pull request by kperrynrel**
## Description

Addresses issue #119, by updating the np.polyfit() function to newer np.polynomial.Polynomial.fit() function.

### CODE REVIEW

File: pvanalytics/util/_fit.py [hunk 1]
```diff
@@ -4,9 +4,10 @@
 
 
 def _quadratic(xs, ys):
-    # fit a quadratic function of `xs` to the data in `ys`
-    coefficients = np.polyfit(xs, ys, 2)
-    return np.poly1d(coefficients)
+    # fit a quadratic function of `xs` to the data in `ys`  
+    coefficients = list(np.polynomial.Polynomial.fit(xs, ys, 2).convert())
```
Comment (cwhanse) [inline pvanalytics/util/_fit.py:8]: I think if you use `polynomial.polyfit` and `flipud` this would be simpler.

Comment (kperrynrel) [reply pvanalytics/util/_fit.py:8]: image

Tried it out and verified same results--thank you for the suggestion @cwhanse !

**Review by cwhanse — APPROVED** [review]

**Comment by kperrynrel** [comment]
@cwhanse what's our policy here on merging PR's in this package, ie are there a certain amount of reviewers I need to check off on this? Or are we good to merge and close now?

**Comment by cwhanse** [comment]
@kperrynrel was just thinking about that. If either Will or I approve and we don't indicate that the other needs to look, then it's OK to merge. If your work is held up by a PR, feel free to nag us about it.

**Comment by kperrynrel** [comment]
Awesome, thanks for the clarification @cwhanse !

**Pull request merged**